<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportGenerator</class>
 <widget class="QWidget" name="ReportGenerator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(234, 254, 255);</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="2" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLineEdit" name="labelRechercher">
            <property name="enabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnRechercher">
            <property name="text">
             <string>Rechercher</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="6" column="0">
         <widget class="QTableView" name="tableMembre">
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QAbstractScrollArea::AdjustIgnored</enum>
          </property>
          <property name="showGrid">
           <bool>true</bool>
          </property>
          <attribute name="horizontalHeaderCascadingSectionResizes">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>160</number>
          </attribute>
          <attribute name="verticalHeaderShowSortIndicator" stdset="0">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderStretchLastSection">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QComboBox" name="comboBox">
          <item>
           <property name="text">
            <string>ID_Livre</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Date_emprunt</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ID_Membre</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QPushButton" name="btnCharger">
          <property name="text">
           <string>Charger la liste</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="styleSheet">
           <string notr="true"> </string>
          </property>
          <property name="text">
           <string>Rapports sur les livres empruntés</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_5"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Statistiques des livres empruntés</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QPushButton" name="btnChargerLivre">
          <property name="text">
           <string>Charger la liste</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QTableView" name="tableLivre">
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QAbstractScrollArea::AdjustIgnored</enum>
          </property>
          <property name="showGrid">
           <bool>true</bool>
          </property>
          <attribute name="horizontalHeaderCascadingSectionResizes">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>160</number>
          </attribute>
          <attribute name="verticalHeaderShowSortIndicator" stdset="0">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderStretchLastSection">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
